src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceStringTable.vb

```diff
@@ -87,6 +87,7 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
             AllowUserToDeleteRows = False
             EditMode = DataGridViewEditMode.EditOnKeystrokeOrF2
             AutoSizeColumnsMode = DataGridViewAutoSizeColumnsMode.Fill
+            AutoSizeRowsMode = DataGridViewAutoSizeRowsMode.AllCells
```

Comment:
To call out the behavior change for reviewers; this means that rows will automatically be sized to fit content - and you can no longer use mouse to resize heights. I'm okay with this.